src/util/util.h

```diff
@@ -146,6 +146,8 @@ enum sssd_exit_status {
 #endif
 #endif
 
+#define to_uint32(val)  ((uint32_t) ((val) & 0x0FFFFFFFF))
```

Comment:
Hi,

I think this is a good idea since it will better meet the intention in case the value is very large.

Since `to_uint32()` is currently only used with `tevent_timeval_current_ofs()` in case the first argument is of type `time_t` I wonder if the code would be easier to read if you would add a wrapper for `tevent_timeval_current_ofs()`, e.g. `sss_tevent_timeval_current_ofs_time_t(time_t t)`?

bye,
Sumit

Reply:
For readability, I created an inline function called as you proposed, using the `UINT_MAX` value when needed.